var/spack/repos/builtin/packages/py-gym/package.py

```diff
@@ -16,6 +16,7 @@ class PyGym(PythonPackage):
     homepage = "https://github.com/openai/gym"
     pypi = "gym/0.18.0.tar.gz"
```

Comment:
```suggestion
    pypi = "gym/gym-0.18.0.tar.gz"
```
No idea how this ever worked. I guess we've been relying on the downloads mirror.

Reply:
First of all: thanks for your time!

As a spack newbie I did not realize about the url issue :-/

I've added a second commit fixing this and the dependencies you mentioned in the other comment (my completely fault not to check that, sorry O:) ).

I've locally rebased my two commits, the commit I used for this PR and the new one fixing the issues and then I've forced a push in my develop branch... but I am not sure if that is the appropriate way of acting (I mean doing that previous rebase with the current state of the develop branch). :-?